### PR TITLE
[Refactor] Fix min_cumulative_compaction_failure_interval_sec misspell

### PR DESCRIPTION
**Pull request by meegoo**
Signed-off-by: meegoo <hujie-dlut@qq.com>

## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [x] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [x] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

**Review by imay — DISMISSED** [review]

File: be/src/common/config.h [hunk 1]
```diff
@@ -310,7 +310,7 @@ CONF_mInt32(repair_compaction_interval_seconds, "600"); // 10 min
 // compaction until this interval passes.
 CONF_mInt64(min_compaction_failure_interval_sec, "120"); // 2 min
 
-CONF_mInt64(min_cmumulative_compaction_failure_interval_sec, "30"); // 30s
+CONF_mInt64(min_cumulative_compaction_failure_interval_sec, "30"); // 30s
```
Comment (imay) [inline be/src/common/config.h:313]: no other references?

Comment (meegoo) [reply be/src/common/config.h:313]: not done yet

**Review by gengjun-git — APPROVED** [review]

**Review by Astralidea — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by meegoo** [comment]
run starrocks_clang-format

**Review by imay — APPROVED** [review]

**Comment by meegoo** [comment]
@mergify backport branch-2.5

**Pull request merged**